policy/mcs

```diff
@@ -138,6 +135,9 @@ mlsconstrain { db_database db_schema db_table db_sequence db_view db_procedure d
 mlsconstrain { db_tuple } { insert relabelto }
 	(( h1 dom h2 ) and ( l2 eq h2 ));
 
+mlsconstrain context contains
+	(( h1 dom h2 ) and ( l1 domby l2 ));
+
```

Comment:
This needs to move up to line 126 since it is not a db constraint. Also, why not have `or ( t1 != mcs_constrained_type )` condition too?

Reply:
I will move it up.

Not sure why it doesn't have `or ( t1 != mcs_constrained_type )`, as I pulled it directly from Fedora's policy as-is. It was added in this nondescript commit: https://github.com/fedora-selinux/selinux-policy/commit/f9ba5d862027233ab129ea021dfa340437d3b08e

Maybe someone else might know.
cc @doverride